### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_Content on Hover or focus : "View Message" tool-tip content is not hoverable using mouse pointer. 

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3464]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.13;#Win32;#A11ySev2;#Benchmark;#FTP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
### 
**Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Hover over "View message" button using mouse.
7. Check whether appeared tooltip content is hoverable using mouse or not.

### **Actual:**
"View Message" tool-tip content is not hoverable using mouse pointer. 

**Similar issue also observed for following scenarios:**

1. Queues-->AddMessage
2. Queues-->DequeueMessage
3. Queues-->ClearQueue

### **Expected:**
"View Message" tool-tip content should be hoverable using mouse pointer. 

### **User Impact:**
Keyboard users as well as users with motor disabilities find it difficult to see the content of the tool tip.

### **Attachment**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-prraha_microsoft_com/ETSvRvAEGZBIos3fzwr84asB2OkNaQIjTK8o3OCmipvtQg?e=eBt0N1

**Comment by JasonYeMSFT** [comment]
@pranshu512 Could you explain what do you mean by "hoverable using mouse pointer"? In browser, tooltip only shows up when you hover on the element and is not hoverable. One of our workaround for these tooltips is that we add the content to their aria-description so users without mouse can still access the content without worrying about hovering over the element. Is there anything missing from the workaround?

**Comment by pranshu512** [comment]
@JasonYeMSFT User should be able to move mouse cursor to the appeared tooltip content and tooltip should not be disappeared when user try to move mouse cursor to the appeared tooltip content

**Comment by JasonYeMSFT** [comment]
I have read through the specification of WCAG 1.4.13. As for now, all the on-hover contents (tooltips) are from the "title" attribute of html, which is completely controlled by the browser. Therefore, the rule doesn't apply to them right now and there is no feasible way for us to change its behavior. I think this issue can be closed.

See "Additional Notes" section of https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html.

**Comment by PoojaNamde** [comment]
Closing the bug as per comment.

Github tags: #Closed;
